lm_eval/tasks/README.md

```diff
@@ -11,6 +11,8 @@
 | [aexams](aexams/README.md) | Tasks in Arabic related to various academic exams covering a range of subjects. | Arabic |
 | [agieval](agieval/README.md) | Tasks involving historical data or questions related to history and historical texts. | English, Chinese |
 | [anli](anli/README.md) | Adversarial natural language inference tasks designed to test model robustness. | English |
+| [arabic_leaderboard_complete](arabic_leaderboard_complete/README.md) | A full version of the tasks focusing on the evaluation of models that reflect the characteristics of Arabic language understanding and comprehension, culture, and heritage. | Arabic |
```

Comment:
```suggestion
| [arabic_leaderboard_complete](arabic_leaderboard_complete/README.md) | A full version of the tasks in the Open Arabic LLM Leaderboard focusing on the evaluation of models that reflect the characteristics of Arabic language understanding and comprehension, culture, and heritage. | Arabic |
```

Reply:
Could we also make more clear that some of these tasks are machine-translated?

Reply:
done